secure_message/resources/messages.py

```diff
@@ -89,12 +89,19 @@ def _try_send_alert_email(message):
                     recipient_email = party_data[0]['emailAddress'].strip()
                     alert_method = AlertViaLogging() if current_app.config['NOTIFY_VIA_GOV_NOTIFY'] == '0' else AlertViaGovNotify()
                     alert_user = AlertUser(alert_method)
-                    alert_user.send(recipient_email, message.msg_id, survey_id=message.survey, party_id=party_data[0]['id'])
+                    personalisation = MessageSend.create_message_url(message.thread_id)
+                    alert_user.send(recipient_email, message.msg_id, personalisation=personalisation,
+                                    survey_id=message.survey, party_id=party_data[0]['id'])
                 else:
                     logger.error('User does not have an emailAddress specified', msg_to=message.msg_to[0])
             # else not testable as fails validation
         return party_data
 
+    @staticmethod
+    def create_message_url(thread_id):
+        url = f'{current_app.config["RAS_FRONTSTAGE_SERVICE"]}secure-message/thread/{thread_id}'
```

Comment:
Can you add the anchor to #latest-message

Reply:
Added anchor to the url